ldes-fragmentisers/ldes-pagination/src/main/java/be/vlaanderen/informatievlaanderen/vsds/config/PaginationProperties.java

```diff
@@ -0,0 +1,9 @@
+package be.vlaanderen.informatievlaanderen.vsds.config;
+
+public class PaginationProperties {
```

Comment:
To be in line with the other fragmenters, i'd call this PaginationConstants

Reply:
I have added PaginationConstants and kept memberlimit in this class like in the timebased fragmentiser